src/SoccerStatistics.Api.Core/DTO/StadiumDTO.cs

```diff
@@ -15,6 +17,6 @@ public class StadiumDTO
         public uint Lighting { get; set; }
         public string Architect { get; set; }
         public bool IsNational { get; set; }
-        public TeamBasicDTO Team { get; set; }
+        public IEnumerable<TeamBasicDTO> Teams { get; set; }
```

Comment:
I think stadium has only 1 representative team. Do we need Ienumerable here?

Reply:
This collection includes teams that play "at home" and do not own the stadium. If necessary, we can add OwnerTeam property.